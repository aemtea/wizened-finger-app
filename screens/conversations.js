import { View } from 'react-native';
import ConversationPreview from '../components/conversationPreview';

const ConversationsScreen = () => {
  return (
    <View>
      <ConversationPreview />
      <ConversationPreview />
      <ConversationPreview />
    </View>
  );
};

export default ConversationsScreen;