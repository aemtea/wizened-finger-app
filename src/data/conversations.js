const conversations = [
  {
    id: 0,
    partyIds: [0, 1]
  }
];

export default conversations;