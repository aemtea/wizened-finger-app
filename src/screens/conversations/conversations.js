import { View } from 'react-native';
import ConversationPreview from '../../components/conversationPreview/conversationPreview';
import styles from './conversations.style';

const ConversationsScreen = ({ navigation }) => {
  return (
    <View style={styles.container}>
      <ConversationPreview navigation={navigation} />
      <ConversationPreview navigation={navigation} />
      <ConversationPreview navigation={navigation} />
    </View>
  );
};

export default ConversationsScreen;