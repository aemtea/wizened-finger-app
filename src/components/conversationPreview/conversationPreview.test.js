import renderer from 'react-test-renderer';
import ConversationPreview from './conversationPreview';
import conversations from '../../data/conversations'

describe('<ConversationPreview />', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<ConversationPreview conversation={conversations[0]}/>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});