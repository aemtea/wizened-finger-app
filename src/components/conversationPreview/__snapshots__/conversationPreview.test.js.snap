// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ConversationPreview /> renders correctly 1`] = `
<View>
  <View
    accessible={true}
    collapsable={false}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "flexDirection": "row",
      }
    }
  >
    <Image
      source={
        Object {
          "uri": "https://i.imgur.com/B1oKCwK.png",
        }
      }
      style={
        Object {
          "height": 72,
          "width": 72,
        }
      }
    />
    <View
      style={
        Object {
          "flexDirection": "column",
        }
      }
    >
      <Text
        style={
          Object {
            "fontWeight": "bold",
          }
        }
      >
        Blaidd the Half-Wolf
      </Text>
      <Text>
        enemy ahead
      </Text>
    </View>
  </View>
</View>
`;
